app/views/govuk_publishing_components/components/metadata/_sentence.html.erb

```diff
@@ -14,7 +14,9 @@
        data-controls="toggle-<%= toggle_id %>"
        data-expanded="false"
        data-toggled-text="<%= t("components.metadata.toggle_less") %>">
-        <%= t("components.metadata.toggle_more", number: remaining.length) %>
+        <%= t("components.metadata.toggle_more",
+          show: t('components.accordion.show'),
```

Comment:
I can see why you've used the existing translation for show, but I think it would be cleaner to give each component its own translation otherwise you're creating a dependency between the two - a dependency makes sense if you're including one in the other but that's not happening here. I know that's going to be a lot of files to update and it will be repetitive but it's the cleaner approach and keeps each component self contained.

Reply:
Actually, perhaps it makes sense to move this translation into "common" and have these two components, plus the accordion pull it from there to avoid repetition but keep it cleaner.

Reply:
Thanks, yep that'll be much tidier. I've now moved a few things under `common`:

- `accordion.show` (used by `accordion`, `related_navigation`, `metadata`)
- `accordion.hide` (only used in `accordion` now but I probably best to keep it together with `show`)
- `metadata.toggle_less` (used by `related_navigation`, `metadata`)
- `metadata.toggle_more` (used by `related_navigation`, `metadata`)